redisinsight/ui/src/pages/vector-search/components/namespaces-browser/NamespacesBrowser.tsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react'
+
+import { ColumnDef, Table } from 'uiSrc/components/base/layout/table'
+import { Row } from 'uiSrc/components/base/layout/flex'
+import { RiIcon } from 'uiSrc/components/base/icons'
+import { Text } from 'uiSrc/components/base/text'
+import {
+  StyledHeader,
+  StyledNamespacesBrowser,
+  StyledRadioGroup,
+  StyledTableContainer,
+  VerticalSeparator,
+} from './styles'
+import { RowRadioButton } from './TableRowRadioButton'
+import { NampespacesBrowserTableDataProps } from './types'
+import { EXAMPLE_DATA } from './data'
+
+export const NAMESPACES_BROWSER_TABLE_COLUMNS = [
+  {
+    id: 'row-selection',
+    size: 10,
+    // We can implement customn selection via a radio button
+    cell: ({ row }) => <RowRadioButton row={row} />,
+    // Or we can use the built-in checkbox selection
+    // cell: ({ row }) => <Table.RowSelectionButton row={row} />,
```

Comment:
By default, Redis UI comes with checkbox row selection, which we can always use if we don't like the "radio" custom implementation.